src/styles/commons.css

```diff
@@ -269,4 +269,13 @@ section.mission {
 */
 .white > label.fr-label {
   color: white;
+}
+
+#whiteColor {
```

Comment:
- Don't use id for css, only for selector (and some very, very specific cases)
- Don't camelCase classes and ids

Here, it should be something like :
```css
.white {

}
```